Dockerfile

```diff
@@ -3,6 +3,6 @@ FROM python:3.6
 WORKDIR /app
 EXPOSE 5000
 COPY ./requirements.txt .
-RUN pip install -r requirements.txt ipdb==0.9.3 --no-cache-dir --disable-pip-version-check
```

Comment:
I guess if filebrowser_s3 is no longer subject to constant change, we don't need this change in the Dockerfile.